tests/src/test/resources/qpid/qdrouterd-with-broker.json

```diff
@@ -110,7 +110,7 @@
       "maxConnections": 40,
       "groups": {
         "Hono": {
-          "users": "Eclipse IoT;Hono;http-adapter,Eclipse IoT;Hono;mqtt-adapter,Eclipse IoT;Hono;amqp-adapter,Eclipse IoT;Hono;coap-adapter,Eclipse IoT;Hono;command-router",
+          "users": "Eclipse IoT;Hono;http-adapter,Eclipse IoT;Hono;mqtt-adapter,Eclipse IoT;Hono;amqp-adapter,Eclipse IoT;Hono;coap-adapter,Eclipse IoT;Hono;command-router,Eclipse IoT;Hono;device-registry",
```

Comment:
I believe that this is not (yet) required as the registries do not support auto-provisioning yet, right?